scripts/pysa_fuzzer/Makefile

```diff
@@ -0,0 +1,31 @@
+# Generate Python files
```

Comment:
Can we rewrite this as a python script?
Sure, Makefiles could do the job here, but there are very limited.
They also force us to use bash, which is a very bad as a programming language.
For instance, the fact that you had to use another python script `find_undetected_files.py` shows the limit of the Makefile.